quipucords/scanner/network/utils.py

```diff
@@ -183,16 +184,17 @@ def raw_facts_template():
     return {fact_name: None for fact_name in get_fact_names()}
 
 
-GEN_KEYFILES_DIRECTORY_PREFIX = "quipucords_ssh_private_key_files_"
-GEN_KEYFILE_PREFIX = "ssh_private_key_file_credential_"
+GEN_KEYFILES_DIRECTORY = "quipucords_generated_ssh_private_keys"
+GEN_KEYFILE_PREFIX = "quipucords_ssh_private_key_credential"
 
 
 def generate_ssh_keyfile(credential, ssh_keyvalue):
     """Generate an ssh private keyfile for the ssh key content specified."""
-    private_keyfile_path = os.path.join(
-        tempfile.mkdtemp(prefix=GEN_KEYFILES_DIRECTORY_PREFIX),
-        f"{GEN_KEYFILE_PREFIX}{credential.get('id')}",
-    )
+    ssh_pkey_dir = os.path.join(tempfile.gettempdir(), GEN_KEYFILES_DIRECTORY)
+    Path(ssh_pkey_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
we can use just pathlib here (I'm almost enabling a ruff check for preferring pathlib XD)

```
    ssh_pkey_dir = Path(tempfile.gettempdir()) / GEN_KEYFILES_DIRECTORY
    ssh_pkey_dir.mkdir(parents=True, exist_ok=True)
```